app/models/gestionnaire.rb

```diff
@@ -45,10 +45,8 @@ def toggle_follow_dossier dossier_id
     nil
   end
 
-  def follow? dossier_id
-    dossier_id = dossier_id.id if dossier_id.class == Dossier
-
-    Follow.where(gestionnaire_id: id, dossier_id: dossier_id).any?
+  def follow?(dossier)
+    followed_dossiers.exists?(dossier.id)
```

Comment:
J'ai creusé. Ça fait la meme requete, donc plutot partant pour du `include`, plus naturel IMO, mais c'est up to you.

```ruby
Dossier.first.commentaires.include?(Commentaire.last)
  Dossier Load (1.4ms)  SELECT  "dossiers".* FROM "dossiers" WHERE "dossiers"."hidden_at" IS NULL ORDER BY "dossiers"."id" ASC LIMIT $1  [["LIMIT", 1]]
  Commentaire Load (0.6ms)  SELECT  "commentaires".* FROM "commentaires" ORDER BY "commentaires"."id" DESC LIMIT $1  [["LIMIT", 1]]
  Commentaire Exists (0.9ms)  SELECT  1 AS one FROM "commentaires" WHERE "commentaires"."dossier_id" = $1 AND "commentaires"."id" = $2 LIMIT $3  [["dossier_id", 112], ["id", 14848], ["LIMIT", 1]]
=> false

irb(main):021:0> Dossier.first.commentaires.exists?(Commentaire.last.id)
  Dossier Load (0.8ms)  SELECT  "dossiers".* FROM "dossiers" WHERE "dossiers"."hidden_at" IS NULL ORDER BY "dossiers"."id" ASC LIMIT $1  [["LIMIT", 1]]
  Commentaire Load (0.6ms)  SELECT  "commentaires".* FROM "commentaires" ORDER BY "commentaires"."id" DESC LIMIT $1  [["LIMIT", 1]]
  Commentaire Exists (0.7ms)  SELECT  1 AS one FROM "commentaires" WHERE "commentaires"."dossier_id" = $1 AND "commentaires"."id" = $2 LIMIT $3  [["dossier_id", 112], ["id", 14848], ["LIMIT", 1]]
=> false
```

```ruby
d = Dossier.first
c = Commentaire.last

d.commentaires.include?(c)
  Commentaire Exists (1.0ms)  SELECT  1 AS one FROM "commentaires" WHERE "commentaires"."dossier_id" = $1 AND "commentaires"."id" = $2 LIMIT $3  [["dossier_id", 112], ["id", 14848], ["LIMIT", 1]]
=> false

d = Dossier.first
c = Commentaire.last

d.commentaires.exists?(c.id)
  Commentaire Exists (0.7ms)  SELECT  1 AS one FROM "commentaires" WHERE "commentaires"."dossier_id" = $1 AND "commentaires"."id" = $2 LIMIT $3  [["dossier_id", 112], ["id", 14848], ["LIMIT", 1]]
=> false
```